src/JustSaying/JustSaying.csproj.DotSettings

```diff
@@ -0,0 +1,4 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Maybe this should be in the `.gitignore`?

Reply:
It looks like this file is intended to be for user settings - I thought it was a project settings file. I've removed it and moved the rule exclusions into code using `// ReSharper disable once CheckNamespace` to avoid the squiggly.